shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/resources/spy.properties

```diff
@@ -0,0 +1,247 @@
+#################################################################
```

Comment:
Consider delete this file, let user add this is better.

Reply:
fine